packages/shared/lib/core/profile/stores/login-progress.store.ts

```diff
@@ -4,10 +4,10 @@ import { ILoginProgress } from '../interfaces'
 
 export const loginProgress = writable<ILoginProgress>({ stepCount: 0, stepMessage: LOGIN_STEPS[0] })
 
-export function incrementLoginProgress(): void {
+export function incrementLoginProgress(incrementBy = 1): void {
```

Comment:
Sorry to be pedantic :sweat_smile:, can we add a `number` type for `incrementBy`?

Reply:
It can be inferred by the default value, but I will add it.